examples/plot_simulate_lfp.py

```diff
@@ -0,0 +1,104 @@
+"""
+===============
+Simulate LFP
+===============
+
+This example demonstrates how to simulate a dipole for evoked-like
+waveforms using HNN-core.
+"""
+
+import os.path as op
+import numpy as np
+
+import matplotlib.pyplot as plt
+import itertools as it
+from glob import glob
+
+from neuron import h
+
+import hnn_core
+from hnn_core.tests import test_network, test_dipole
+from hnn_core.network import Spikes, read_spikes
+from hnn_core import lfp    
+
+import matplotlib.pyplot as plt
+
+from hnn_core import JoblibBackend
+from hnn_core import simulate_dipole, read_params, Network, read_spikes, viz
+
+hnn_core_root = op.dirname(hnn_core.__file__)
+
+params_fname = op.join(hnn_core_root, 'param', 'default.json')
+params = read_params(params_fname)
+
+net = Network(params)
+
+###############################################################################
+# Let us import hnn_core
+
+from hnn_core.pyramidal import L5Pyr
+from hnn_core.network_builder import load_custom_mechanisms
+
+load_custom_mechanisms()
+cell = L5Pyr()
+soma_pos = [cell.get3dinfo()[idx][0] for idx in range(3)]
+apical_trunk_pos = [cell.get3dinfo()[idx][1] for idx in range(3)]
+
+num_elec = 5
+x_width = [-100,100]
+y_width = [-100,100]
+
+z_pos = 0
+elec_grid = np.array([[c,r,z_pos] for c in np.linspace(x_width[0], x_width[1], num_elec) 
+            for r in np.linspace(y_width[0], y_width[1], num_elec)])
+
+h.load_file("stdgui.hoc")
+h.cvode_active(1)
+
+ns = h.NetStim()
+ns.number = 1
+ns.start = 100
+ns.interval = 50.0
+ns.noise = 0.  # deterministic
+
+nc = h.NetCon(ns, cell.synapses['apicaltuft_ampa'])
+nc.weight[0] = 0.001
+
+h.tstop = 500.0
+lfp_rec = {'lsa': ([], []), 'psa': ([], [])}
+for method in ['lsa', 'psa']:
```

Comment:
I think this script could go into @cjayb 's repo: https://github.com/cjayb/hnn-core-examples ? Because it runs pretty slow ... could be a nice place to host some of the longer examples